pajbot/apiwrappers/twitch/helix.py

```diff
@@ -233,3 +233,13 @@ def bulk_get_user_basics_by_login(self, logins):
             else None
             for user_data in bulk_user_data
         ]
+
+    def get_user_data_by_client_id_and_access_token(self, client_id, access_token):
```

Comment:
method name should start with `fetch` instead of `get` since it never uses a cache of any sort, in line with how the rest of the methods in this api wrapper are named (e.g. `fetch_all_subscribers`)